reposcan/database/cvemap_store.py

```diff
@@ -44,7 +44,10 @@ def _import_cves(self, to_import, cve_data):
                                                         cvss3_score, cvss3_metrics, iava,
                                                         redhat_url, secondary_url, source_id,
                                                         cvss2_score, cvss2_metrics)
-                                              values %s returning id, name""",
+                                              values %s on conflict on constraint cve_name_key
+                                              do update set source_id = (select id from cve_source
```

Comment:
this doesn't update any other column than source_id, when for example description changes in meanwhile, you need to sync twice to get it correctly synced, please make sure everything is updated on first sync, rather not copy update code here but re-use `_update_cves` method